### PR TITLE
Feature/us4556 create group saves category

### PR DESCRIPTION
**Pull request by ilpadre**

### CODE REVIEW

File: Gateway/MinistryPlatform.Translation/Repositories/AttributeRepository.cs [hunk 1]
```diff
@@ -3,23 +3,24 @@
...
 using MinistryPlatform.Translation.Repositories.Interfaces;
-using MpAttribute = MinistryPlatform.Translation.Models.MpAttribute;
 
 namespace MinistryPlatform.Translation.Repositories
 {
     public class AttributeRepository : BaseRepository, IAttributeRepository
     {
         private readonly IMinistryPlatformService _ministryPlatformService;
-            
+        private readonly int attributesByTypePageViewId = Convert.ToInt32(AppSettings("AttributesPageView"));
```
Comment (jpc11623) [inline Gateway/MinistryPlatform.Translation/Repositories/AttributeRepository.cs]: Should probably have an underscore in front of these: e.g. _attributesByTypePageViewId.

File: Gateway/MinistryPlatform.Translation/Repositories/AttributeRepository.cs [hunk 2]
```diff
@@ -3,23 +3,24 @@
...
+        private readonly int attributesPageId = Convert.ToInt32(AppSettings("Attributes"));
+
         public AttributeRepository(IMinistryPlatformService ministryPlatformService, IAuthenticationRepository authenticationService, IConfigurationWrapper configurationWrapper)
             : base(authenticationService, configurationWrapper)
         {
             _ministryPlatformService = ministryPlatformService;
         }
```
Comment (jpc11623) [inline Gateway/MinistryPlatform.Translation/Repositories/AttributeRepository.cs]: Remove white space

Comment (jpc11623) [inline Gateway/MinistryPlatform.Translation/Repositories/AttributeRepository.cs]: Extra white space

File: Gateway/crds-angular/Controllers/API/AttributeController.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
```
Comment (JoeKer1) [inline Gateway/crds-angular/Controllers/API/AttributeController.cs]: This file should be removed.

File: Gateway/crds-angular/Services/AttributeService.cs [hunk 4]
```diff
@@ -49,6 +49,25 @@ public AttributeDTO ConvertAttributeToAttributeDto(MpAttribute attribute)
             return attributeDto;
         }
 
+        public int CreateOrUpdateAttributes(List<AttributeDTO> attributes, int attributeType)
+        {
```
Comment (JoeKer1) [inline Gateway/crds-angular/Services/AttributeService.cs]: This method can be removed. It was never used.

**Comment by jpc11623** [comment]
There is some tech debt - need to move the code back to group or attribute service and out of the repo layer. Also need to add a test. Spoke with Scrummy Bears about this, and it will be addressed in the next refactor.

**Pull request merged**